models/dodo/ethereum/dodo_ethereum_trades.sql

```diff
@@ -0,0 +1,348 @@
+(lower('0xc9f93163c99695c6526b799ebca2207fdf7d61ad'), 'USDT', 'USDC', lower('0xdac17f958d2ee523a2206206994597c13d831ec7'), lower('0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48')),
+(lower('0xd4a36b0acfe2931cf922ea3d91063ddfe4aff01f'), 'sUSD', 'USDT', lower('0x57ab1ec28d129707052df4df418d58a2d46d5f51'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7')),
+(lower('0x8876819535b48b551c9e97ebc07332c7482b4b2d'), 'DODO', 'USDT', lower('0x43dfc4159d86f3a37a5a4b3d4580b888ad7d4ddd'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7')),
+(lower('0x9d9793e1e18cdee6cf63818315d55244f73ec006'), 'FIN', 'USDT',  lower('0x054f76beed60ab6dbeb23502178c52d6c5debe40'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7')),
+(lower('0x94512fd4fb4feb63a6c0f4bedecc4a00ee260528'), 'AAVE', 'USDC', lower('0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9'), lower('0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48')),
+(lower('0x85f9569b69083c3e6aeffd301bb2c65606b5d575'), 'wCRESt','USDT',lower('0xa0afaa285ce85974c3c881256cb7f225e3a1178a'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7')),
+(lower('0x3058ef90929cb8180174d74c507176cca6835d73'), 'DAI', 'USDT',  lower('0x6b175474e89094c44da98b954eedeac495271d0f'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7')),
+(lower('0xd84820f0e66187c4f3245e1fe5ccc40655dbacc9'), 'sUSD', 'USDT', lower('0x57ab1ec28d129707052df4df418d58a2d46d5f51'), lower('0xdac17f958d2ee523a2206206994597c13d831ec7'))
+)
+
+,dexs AS 
+    (SELECT
+        dexs.block_time,
+        erc20a.symbol AS token_a_symbol,
+        erc20b.symbol AS token_b_symbol,
+        token_a_amount_raw / POW(10 ,erc20a.decimals) AS token_a_amount,
+        token_b_amount_raw / POW(10 ,erc20b.decimals) AS token_b_amount,
+        project,
+        version,
+        category,
+        coalesce(trader_a, tx.`from`) as trader_a, -- subqueries rely on this COALESCE to avoid redundant joins with the transactions table
+        trader_b,
+        token_a_amount_raw,
+        token_b_amount_raw,
+        coalesce(
+            usd_amount,
+            token_a_amount_raw / POW(10 , (CASE token_a_address WHEN '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' THEN 18 ELSE pa.decimals END)) * (CASE token_a_address WHEN '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' THEN  pa.price ELSE pa.price END),
+            token_b_amount_raw / POW(10 ,(CASE token_b_address WHEN '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' THEN 18 ELSE pb.decimals END)) * (CASE token_b_address WHEN '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' THEN  pb.price ELSE pb.price END)
+        ) as usd_amount,
+        token_a_address,
+        token_b_address,
+        exchange_contract_address,
+        tx_hash,
+        tx.`from` as tx_from,
+        tx.`to` as tx_to,
+        trace_address,
+        evt_index,
+        row_number() OVER (PARTITION BY project, tx_hash, evt_index, trace_address ORDER BY version, category) AS trade_id
```

Comment:
this column is no longer necessary. please remove from select statements, unique keys in config, schema file and anywhere else applicable

Reply:
nvm, i got this covered in latest commit

Reply:
Okay. Let me do that asap.